subsystems/mvdm/ntvdm/bios/vidbios.c

```diff
@@ -3178,6 +3467,36 @@ VOID WINAPI VidBiosVideoService(LPWORD Stack)
             break;
         }
 
+        /* Write Pixel At */
+        case 0x0C:
+        {
+            BYTE Page = getBH();
+
+            /* Validate page (0xFF = current video page) */
+            if (Page == 0xFF)
+                Page = Bda->VideoPage;
+            else if (Page >= BIOS_MAX_PAGES)
+                break;
+
+            VidBiosSetPixel(getCX(), getDX(), Page, getAL());
+            break;
+        }
+
+        /* Read Pixel At */
+        case 0x0D:
+        {
+            BYTE Page = getBH();
+
+            /* Validate page (0xFF = current video page) */
```

Comment:
If we believe Ralf Brown
http://www.ctyme.com/intr/rb-0105.htm
"BH is ignored if the current video mode supports only one page."